ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6620,3 +6620,19 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+OAQM:
+  description: "Outside air quality monitoring (particulates and gases)"
+  is_abstract: true
+  uses:
+  - outside_air_pm1pt0_density_sensor
+  - outside_air_pm2pt5_density_sensor
+  - outside_air_pm10pt0_density_sensor
+  - outside_air_no_density_sensor
+  - outside_air_no2_density_sensor
+  - outside_air_so2_density_sensor
+  - outside_air_co_concentration_sensor
```

Comment:
You are mixing `density` and `concentration` measurements; please confirm the units of measurement for each sensor.

Reply:
we are being informed that the measurements for `NO`, `NO2`, `SO2` are in `micrograms per cubic meters` which is also what we see in telemetry message. `CO` is "ppm" so I used concentration for this field.

Reply:
Thank you for confirming!